hypothetical-use-cases/trade-certificates/examples/resolver.js

```diff
@@ -0,0 +1,12 @@
+const didDocs = require("./didDocs");
```

Comment:
duplicate of `hypothetical-use-cases/basic/resolver.js`

Reply:
I have a feeling we may remove these later, for now its helpful to have the dependencies separated.